include/seqan3/alignment/pairwise/alignment_configurator.hpp

```diff
@@ -489,13 +493,13 @@ struct alignment_configurator
         // Temporarily we will use the new and the old alignment implementation in order to
         // refactor step-by-step to the new implementation. The new implementation will be tested in
         // macrobenchmarks to show that it maintains a high performance.
-
+        constexpr bool more_than_score = traits_t::compute_end_positions ||
+                                         traits_t::compute_begin_positions ||
+                                         traits_t::compute_sequence_alignment;
         // Use old alignment implementation if...
-        if constexpr (traits_t::is_local ||                                          // it is a local alignment,
-                      traits_t::is_debug ||                                          // it runs in debug mode,
-                      traits_t::compute_sequence_alignment ||                        // it computes more than the begin position.
-                     (traits_t::is_banded && traits_t::compute_begin_positions) ||   // banded && more than end positions.
-                     (traits_t::is_vectorised && traits_t::compute_end_positions))   // simd and more than the score.
+        if constexpr (traits_t::is_local ||                                                            // it is a local alignment,
+                     (traits_t::is_banded && (more_than_score || traits_t::compute_end_positions)) ||  // banded
```

Comment:
Hmm

```
  more_than_score
= traits_t::compute_end_positions || traits_t::compute_begin_positions || traits_t::compute_sequence_alignment

  more_than_score || traits_t::compute_end_positions
= traits_t::compute_end_positions || traits_t::compute_begin_positions || traits_t::compute_sequence_alignment || traits_t::compute_end_positions
= traits_t::compute_end_positions || traits_t::compute_end_positions || traits_t::compute_begin_positions || traits_t::compute_sequence_alignment
= traits_t::compute_end_positions || traits_t::compute_begin_positions || traits_t::compute_sequence_alignment
= more_than_score
```

It seems like someone changed the behaviour of `more_than_score` at some point in time :)

Reply:
Just a stupid question: Activating the new alignment for `more_than_score` could have done in a separate PR for quite some time, right?

Reply:
This was activated for most alignment but not for all. For example local still uses the old alignment or banded and vectorised alignment. Though banded also allows end positions but not more. This comes in future PRs.